src/components/tutorial/StartButtonPointer.vue

```diff
@@ -0,0 +1,30 @@
+<template>
```

Comment:
Yeah all these template files are mostly the same exactly the same -- can you combine and parametrize? 

If not possible before tomorrow ,let it be. When we open source it, we can do it 👍🏾